x-pack/solutions/observability/plugins/slo/docs/openapi/slo/components/schemas/bulk_purge_rollup_request.yaml

```diff
@@ -0,0 +1,32 @@
+title: Bulk Purge Rollup data request
+description: >
+  The bulk purge rollup data request takes a list of SLO ids and a purge policy, then deletes the rollup data according to the purge policy.
+  This API can be used to remove the staled data of an instance SLO that no longer get updated.
+type: object
+required:
+  - list
+  - purgePolicy
+properties:
+  list:
+    description: An array of slo ids
+    type: array
+    items:
+      type: string
+      description: The SLO Definition id
+      example: 8853df00-ae2e-11ed-90af-09bb6422b258
+  purgePolicy:
+    description: Policy that dictates which SLI documents to purge based on age
+    type: object
```

Comment:
Use a oneOf schema definition to indicate it is either a { purgeType: 'fixed_age', age: string } or { purgeType: 'fixed_time', timestamp: date string }. There are some example in other requests